.github/actions/check-size/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Check size of a policy package
+description: Checks the disk size of a policy package
+
+inputs:
+  package:
+    description: the name of the package
+    required: true
+  limit:
+    description: the maximum size of the package in kB
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Check policy package size
+      env:
+        package: ${{ inputs.package }}
+        limit: ${{ inputs.limit }}
+      run: |
+        failed=false
+
+        actual=$(du -k "$package" | cut -f1)
+
+        if [ "$actual" -gt "$limit" ]; then
+            echo "Size of '$package' of $actual kB is greater than the limit of $limit kB"
+            failed=true
+        else
+            echo "Size of '$package' of $actual kB is at or under the limit of $limit kB"
+        fi
+
+        if [ "$failed" == true ]
```

Comment:
bad syntax here, should be `if [ "$failed" == true ]; then`

Reply:
Strangely, actually does work like this. I'll keep an eye on it and if bad, will fix.

Reply:
Done! (although it was working fine like this)